package-lock.json

```diff
@@ -7165,7 +7165,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
Could you revert these changes since there's no accompanying `package.json` changes?

Reply:
Sure. This probably got changed due to different NPM versions.

Reply:
Will update this while updating the version of `mattermost-redux` commit in `package.json`

Reply:
It might also have been something weird like `npm install` vs `npm add` with a new version of mattermost-redux. `npm` is weird like that sometimes